lib/merkl.ts

```diff
@@ -64,10 +77,52 @@ export const claimMerklRewards = async (smartAccountClient: SmartAccountClient,
     },
   ]
 
-  await executeTransactions(
-    smartAccountClient,
-    transactions,
-    'Failed to claim Merkl rewards',
-    chain,
-  )
+  const amount = amounts.reduce((acc, amount) => acc + amount, 0n)
+  const formattedAmount = formatUnits(amount, 6)
+  // Track only successful claim rewards
+  const status = TransactionStatus.SUCCESS
+
+  const result = await trackTransaction(
+    {
+      type: TransactionType.MERKL_CLAIM,
```

Comment:
@MusabShakeel576 Please also add this type to enum `ActivityType` in the backend

Reply:
@LiorAgnin added
https://github.com/Solid-Money/solid-backend/pull/368